packages/framework/src/Component/Money/Money.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Money;
+
+use Litipk\BigNumbers\Decimal;
+
+/**
+ * Immutable object for representing a monetary value
+ * Uses decimal numbers with specifiable number of decimal places
+ */
+class Money
```

Comment:
We have discussed this with @PetrHeinz and we want to postpone this decision until end of implementation of whole US. We might implement factory for this + we do not use final in framework yet and it might be in conflict with some sniffs.

Reply:
Money class marked as final in cc04c5c54